code/ExampleCommands.cs

```diff
@@ -39,4 +39,43 @@ public static void NoclipCommand()
             }
         }
     }
+
+    [ConCmd.Server("damageme", Help = "damage me")]
+    public static void DamageCommand(int amount = 10)
+    {
+        var client = ConsoleSystem.Caller;
+
+        if (client != null && client.Pawn is PlayerBase ply && ply.ActiveChild is WeaponBase weapon)
+        {
+            var damageInfo = new DamageInfo();
+            damageInfo.Damage = amount;
+            damageInfo.Weapon = weapon;
+
+            ply.TakeDamage(damageInfo);
+        }
+    }
+
+    [ConCmd.Server("swb_attachment_equip", Help = "Equips an attachment by name")]
+    public static void EquipAttachmentCMD(string name, bool enabled)
+    {
+        var client = ConsoleSystem.Caller;
+
+        if (client != null)
+        {
+            var player = client.Pawn as PlayerBase;
+
+            if (player.ActiveChild is not WeaponBase activeWeapon) return;
```

Comment:
Could we cast to the player interface here and move the command to `swb_base/Commands.cs`, the test commands are really just for testing in the demo gamemode but attachments are part of the base, inseparable.

Reply:
But that means that by using the weapon base, anybody can just call `swb_attachment_equip` and attach an attachment. What if I don't want players to be able to equip attachments? What if they need to buy them or something.

This would force me to then go into `swb_base/Commands.cs` and make modifications to enforce access to attachments.

Reply:
I will think of a solution to this for a future update, there should be a way to disable that.